src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncerTest.java

```diff
@@ -835,12 +853,14 @@ public final void testDetermineNewLeasesToCreateDoesntIgnoreClosedShardWithEmpty
     /**
      * Test CheckIfDescendantAndAddNewLeasesForAncestors (initial position Latest)
      * Shard structure (each level depicts a stream segment):
-     * 0 1 2 3 4 5- shards till epoch 102
-     * \ / \ / | |
-     * 6 7 4 5- shards from epoch 103 - 205
-     * \ / | /\
-     * 8 4 9 10 - shards from epoch 206 (open - no ending sequenceNumber)
+     * 0 1 2 3 4   5- shards till epoch 102
+     * \ / \ / |   |
+     *  6   7  4   5- shards from epoch 103 - 205
+     *   \ /   |  /\
+     *    8    4 9  10 - shards from epoch 206 (open - no ending sequenceNumber)
      * Current leases: (3, 4, 5)
+     * Initial position: LATEST
+     * Leases to create: (2, 6)
```

Comment:
Thanks for fixing these :)

Reply:
Yeah old diagrams are unbelievably difficult to decipher.